src/web/components/MostViewed/MostViewedRight/MostViewedRightItem.tsx

```diff
@@ -106,6 +105,11 @@ export const MostViewedRightItem = ({ trail }: Props) => {
                             size="tiny"
                             showUnderline={isHovered}
                             {...itemProps}
+                            link={{
```

Comment:
Refactored `SmallHeading` once again to encapsulate all the link related properties inside a `link` object instead of using separate props.